lms/djangoapps/discussion/django_comment_client/base/views.py

```diff
@@ -97,6 +109,41 @@ def track_forum_event(request, event_name, course, obj, data, id_map=None):
     with tracker.get_tracker().context(event_name, context):
         tracker.emit(event_name, data)
 
+    SIGNAL = TRACKING_LOG_TO_EVENT_MAPS.get(event_name, None)
+    if SIGNAL is not None:
+        SIGNAL.send_event(
+            thread=DiscussionThreadData(
+                anonymous=data.get('anonymous'),
+                anonymous_to_peers=data.get('anonymous_to_peers'),
+                body=data.get('body'),
+                category_id=data.get('category_id'),
+                category_name=data.get('category_name'),
+                commentable_id=data.get('commentable_id'),
+                group_id=data.get('group_id'),
+                id=data.get('id'),
+                team_id=data.get('team_id'),
+                thread_type=data.get('thread_type'),
+                title=data.get('title'),
+                title_truncated=data.get('title_truncated'),
+                truncated=data.get('truncated'),
+                url=data.get('url'),
+                discussion=data.get('discussion'),
+                user_course_roles=data.get('user_course_roles'),
+                user_forums_roles=data.get('user_forums_roles'),
+                user=UserData(
+                    pii=UserPersonalData(
+                        username=user.username,
+                        email=user.email,
+                        name=user.profile.name,
+                    ),
+                    id=user.id,
+                    is_active=user.is_active,
+                ),
+                course_id=str(course.id),
+                options=data.get('options'),
+            )
+        )
```

Comment:
can you add unittests for this? As we've done in several other PR(s): https://github.com/openedx/edx-platform/pull/32599/files#diff-11fef21cda98f69e1fb1129e50f0813cd18253bf52adff29f45bc8424446eabeR945-R1043